@kalyanachakravarthikarri-az , please fix duplicated schema issue that caused Go sdk generation failure.
```
error   | PreCheck/DuplicateSchema | Duplicate Schema named 'MultiUserAuthorization' (4 differences):
cmderr	[generator automation-v2] [ERROR] - type: "object" => "string"
cmderr	[generator automation-v2] [ERROR] - properties: {"multiUserAuthorization":{"x-ms-enum":{"name":"MultiUserAuthorization","modelAsString":true},"$ref":"#/components/schemas/MultiUserAuthorization"}} => <none>
cmderr	[generator automation-v2] [ERROR] - enum: <none> => ["Invalid","Enabled","Disabled"]
cmderr	[generator automation-v2] [ERROR] - x-ms-enum: <none> => {"name":"MultiUserAuthorization","modelAsString":true}; This error can be *temporarily* avoided by using the 'modelerfour.lenient-model-deduplication' setting.  NOTE: This setting will be removed in a future version of @autorest/modelerfour; schemas should be updated to fix this issue sooner than that.
cmderr	[generator automation-v2] [ERROR] fatal   | Error: 1 errors occured -- cannot continue.
```

--- a/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/vaults.json
+++ b/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/vaults.json
@@ -1217,10 +1217,10 @@
     "MultiUserAuthorization": {
       "description": "MUA Settings of vault",
       "enum": [
-            "Invalid",
-            "Enabled",
-            "Disabled"
-          ],
+        "Invalid",
+        "Enabled",
+        "Disabled"
+      ],
       "type": "string",
       "readOnly": true,
       "x-ms-enum": {

--- a/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/vaults.json
+++ b/specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/vaults.json
@@ -1216,20 +1216,16 @@
     },
     "MultiUserAuthorization": {
       "description": "MUA Settings of vault",
-      "type": "object",
-      "properties": {
-        "multiUserAuthorization": {
-          "enum": [
+      "enum": [
             "Invalid",
             "Enabled",
             "Disabled"
           ],
-          "type": "string",
-          "x-ms-enum": {
-            "name": "MultiUserAuthorization",
-            "modelAsString": true
-          }
-        }
+      "type": "string",
+      "readOnly": true,
+      "x-ms-enum": {
+        "name": "MultiUserAuthorization",
+        "modelAsString": true
       }
     },
     "MonitoringSettings": {

